scripts/averaging/create_TEM_files.py

```diff
@@ -50,45 +75,60 @@ def create_TEM_files(adf):
         #If dictionary is empty, then there are no observations, so quit here:
         if not var_obs_dict:
             print("No observations found to plot against, so no TEM will be generated.")
-            return
+            pass
```

Comment:
If this is now a `pass` statement then I might vote to change the `print` statement to something like:

`"No observations found to plot against, so no obs-based TEM file will be generated".`

This is assuming that the standard CAM case TEM files can still be generated without the obs data.  If not then this should probably be turned back into a `return` statement.

Reply:
That was my intention, if the obs don't exist, the test case TEM plots should still be generated. Thanks